pkgs/by-name/sl/slimevr/gui-no-git.patch

```diff
@@ -14,8 +14,9 @@ index c9eb27be..4b3757f0 100644
 +const commitHash = "";
 +const versionTag = "@version@";
```

Comment:
I just noticed, this is missing the `v` prefix because the version we give is just the semver, should we add it?